x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -503,43 +503,62 @@ export const getEndpointConsoleCommands = ({
 
   switch (agentType) {
     case 'sentinel_one':
-      return adjustCommandsForSentinelOne({ commandList: consoleCommands });
+      return adjustCommandsForVendor({
+        commandList: consoleCommands,
+        vendor: 'sentinel_one',
+        featureFlags: {
+          isHostIsolationEnabled: featureFlags.responseActionsSentinelOneV1Enabled,
+          isGetFileFeatureEnabled: featureFlags.responseActionsSentinelOneGetFileEnabled,
+        },
+      });
+    case 'crowdstrike':
+      return adjustCommandsForVendor({
+        commandList: consoleCommands,
+        vendor: 'crowdstrike',
+        featureFlags: {
+          isHostIsolationEnabled: featureFlags.responseActionsCrowdstrikeManualHostIsolationEnabled,
+        },
+      });
     default:
       // agentType === endpoint: just returns the defined command list
       return consoleCommands;
   }
 };
 
 /** @private */
-const adjustCommandsForSentinelOne = ({
+const adjustCommandsForVendor = ({
   commandList,
+  vendor,
+  featureFlags,
 }: {
   commandList: CommandDefinition[];
+  vendor: 'sentinel_one' | 'crowdstrike';
```

Comment:
Can you change this to:

```
vendor: Exclude<ResponseActionAgentType, 'endpoint'>
```

Also: we been using `agentType` in all other areas of the code. Consider renaming the property to `agentType` to remain consistent.

Reply:
Yes, thanks for catching this, not sure how I came up with `vendor` ... ;p